src/mapgen.cpp

```diff
@@ -2155,8 +2190,11 @@ void jmapgen_objects::check( const std::string &oter_name ) const
  * (set|line|square)_(ter|furn|trap|radiation); simple (x, y, int) or (x1,y1,x2,y2, int) functions
  * TODO: optimize, though gcc -O2 optimizes enough that splitting the switch has no effect
  */
-bool jmapgen_setmap::apply( const mapgendata &dat, int offset_x, int offset_y ) const
+bool jmapgen_setmap::apply( const mapgendata &dat, int offset_x, int offset_y, mission *miss ) const
 {
+    if( miss )
+        ;
```

Comment:
This if weird. You could simply omit `miss` in the signature. Or is it a leftover of something?

Reply:
leftovers, thanks for catching it

Reply:
There are quite a few of these, and they trigger warnings.